ui/src/components/progress.tsx

```diff
@@ -10,13 +10,18 @@ const Progress = React.forwardRef<
 >(({ className, value, ...props }, ref) => (
 	<ProgressPrimitive.Root
 		ref={ref}
-		className={cn('relative h-2 w-full overflow-hidden rounded-full bg-white', className)}
+		className={cn('bg-primary relative h-full w-full overflow-hidden rounded-full p-1', className)}
```

Comment:
I'm not sure if the `h-full` is what we want here, but we can adapt it if we use the progress bar somewhere else...